{"name": "Unsuccessful login", "status": "passed", "steps": [{"name": "Error message is visible", "status": "passed", "start": 1737700076766, "stop": 1737700076766}], "parameters": [{"name": "username", "value": "''"}, {"name": "password", "value": "'secret_sauce'"}, {"name": "driver", "value": "'Edge'"}], "start": 1737700076315, "stop": 1737700076862, "uuid": "bf99c5fe-5b7a-4d14-8c0b-2a4129b590ea", "historyId": "e664392ae8e96150bc26ed53191a48bb", "testCaseId": "b876685be5aab7257716ff999cc789c0", "fullName": "Tests.test_Login.TestLogin#test_1", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az1704-590"}, {"name": "thread", "value": "1930-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}